pkg/generators/snippets/constructor.go

```diff
@@ -5,14 +5,12 @@ import (
 	"k8s.io/gengo/types"
 )
 
-func GenerateEmptyConstructor(t *types.Type) (string, generator.Args) {
-	args := generator.Args{
-		"type": t,
-	}
+func GenerateEmptyConstructor(t *types.Type, pointerReceiver bool) (string, generator.Args) {
```

Comment:
For `GenerateEmptyConstructor`, I left the pointerReceiver arg in as an option for flexibility in case there's a point in future where we want to generate pointer receivers for types without ObjectMeta.